2.crud_queries/tasks.md

```diff
@@ -1,38 +1,82 @@
 1. Create a database named `blog`.
 
+use blog
+
 2. Create a collection called 'articles'.
 
+db.createCollection('articles')
+
 3. Insert multiple documents(at least 3) into articles. It should have fields
 
+db.articles.insertMany([{title: 'GOT-1', description: 'A', author: {name: 'A', age: 65}, tags: ['A']},
+{title: 'GOT-2', description: 'B', author: {name: 'B',  age: 66}, tags: ['B']},
+ {title: 'GOT-3', description: 'C', author: {name: 'C', age: 67}, tags: ['C']}])
+
 4. Find all the articles using `db.COLLECTION_NAME.find()`
 
+db.articles.find()
+
 5. Find a document using _id field.
 
+db.articles.find({ "_id" : ObjectId("5d9cb039b43283ca02645914")})
+
+
 6. Find documents using title and author's name field.
 
+db.articles.find({title: 'GOT-1'})
+db.articles.find({title: 'GOT-2'})
+db.articles.find({title: 'GOT-3'})
+
```

Comment:
I mean to say that find documents using title and author's name both at the same time.
```js
db.articles.find({title: 'GOT-1', 'author.name': 'A'})
```
Seperating queries using commas defaults to and operation.